src/Utils/TmpDirectoryCreator.php

```diff
@@ -59,14 +65,25 @@ public function __construct(Filesystem $fileSystem)
         $this->fileSystem = $fileSystem;
     }
 
-    public function createAndGet(string $tempDir): string
+    public function createAndGet(string $tmpDir): string
     {
-        if (!$this->path) {
-            $path = sprintf('%s/%s', $tempDir, self::BASE_DIR_NAME);
+        Assert::true(
+            Path::isAbsolute($tmpDir),
+            sprintf(
+                'Expected the temporary directory passed to be an '
+                . 'absolute path. Got "%s"',
```

Comment:
do we really need concatenating here?

Reply:
hm not sure if I did 80 out of habit, will check